client/app/styles/tip.css

```diff
@@ -23,3 +24,36 @@
 .fieldAnswer {
   margin-bottom: 10px;
 }
+
+.SubmissionInfoSection {
+  background-color: #eaebe2;
+  border-radius: 2%;
+}
+
+.SubmissionContentSection {
+  background-color: #e5fecd;
+  border-radius: 2%;
+}
+
+.left {
+  text-align: right;
+  padding: 0.5%;
+  width: 30%;
+}
+.right {
+  text-align: left;
+  padding: 0.5%;
+  width: 65%;
```

Comment:
we are using bootstrap grid system for a reason that is that it that makes things responsive. avoid using of percentages directly.